walkthroughs/3a-low-code-api-development/walkthrough.adoc

```diff
@@ -189,26 +235,46 @@ A *Path* represents an API endpoint/operation and the associated parameters requ
 
 . Click the orange *POST* icon.
 
-. Enter "Greet with name" in the *Summary* field.
+. Enter the following in the *Summary* field:
++
+----
+Greet with name
+----
 
-. Enter "greetname" in the *Operation ID* field.
+. Enter the following in the *Operation ID* field:
++
+----
+greetname
+----
 
-. In the *Request Body* section, choose the *Name* type that you created earlier.
+. In the *Request Body* click *Add a request body* and choose the *Name* type that you created earlier.
 
 . In the *Responses* section click *Add response*.
-* In the modal that appears choose *200 OK* as the dropdown option.
-* Click *Add*.
-* Click *No description* beside the *200 OK* response and enter "Greeting response" in the *Description* field.
+.. In the modal that appears choose *200 OK* as the dropdown option.
+.. Click *Add*.
+.. Click *No description* beside the *200 OK* response and enter the following in the *Description* field:
++
+----
+Greeting response
+----
 
 . Click *Save* in the top right corner of the UI to save your work and be directed back to the *New Integration* wizard.
 
 . Click *Next* on the *New Integration* wizard screen.
 
-. When prompted to *Give this integration a name* enter:
-* "greeting api" in the *Integration Name* field
-* "my greeting api" in the *Description* field
+. When prompted to *Give this integration a name*:
+.. Enter the following in the *Integration Name* field:
++
+----
+greeting api
+----
+.. Enter the following in the *Description* field:
++
+----
+my greeting api
+----
 
-. Click *Save and Continue* to save your API design and move onto implementation.
+. Click *Save and Continue* to save your API design.
```

Comment:
OK so after this there's a subsequent screen now that has a "Create Flow" button that we need to click. This is instead of the old flow of `On the Choose operation screen click Greet with name to open the Integration Flow Editor.`

Reply:
i.e `On the subsequent screen click the Create Flow button`

Reply:
outdated